test/builder-api/src/misc.js

```diff
@@ -39,7 +39,8 @@ describe('Miscellanenous API', function () {
   });
 
   describe('Retrieving reverse dependencies', function () {
-    it('returns all reverse dependencies for an origin and package name', function (done) {
+    // JAH: ensure this works after rdep refactoring
```

Comment:
A general note for reviewers.  Anywhere you see my initials (JAH) that's something I intend to resolve appropriately before finalizing a PR.  Not saying one will never be committed but if you see JAH somewhere its a sign that there's something about that spot I'm still working on somehow.

Reply:
This has been addressed. Still a good note about my use of my initials.